core/src/navigation/TopNav.html

```diff
@@ -22,8 +22,25 @@
       on:handleClick="{handleClickExternal}"
     />
   </div>
+  {#if globalSearchConfig && isGlobalSearchCentered}
+  <div class="lui-global-search">
+    <GlobalSearchCentered
+      globalSearchConfig="{globalSearchConfig}"
+      bind:isSearchFieldVisible
+      on:toggleSearch
+      on:handleSearchNavigation
+      bind:searchResult
+      bind:displaySearchResult
+      bind:displayCustomSearchResult
+      bind:inputElem
+      bind:luigiCustomSearchRenderer__slot
+      on:closeSearchResult
+    />
+  </div>
+  {/if}
   <div class="fd-shellbar__group fd-shellbar__group--actions">
-    {#if !authorizationEnabled || isLoggedIn} {#if isGlobalSearchAvailable}
+    {#if !authorizationEnabled || isLoggedIn} {#if globalSearchConfig} {#if
+    !isGlobalSearchCentered}
```

Comment:
```suggestion
    {#if !authorizationEnabled || isLoggedIn} {#if
    !isGlobalSearchCentered}
```
It seems `isGlobalSearchCentered` already included `{#if globalSearchConfig}` condition. Do we need to check it twice?